tutorials/artifactory-gcs-setup/index.md

```diff
@@ -148,7 +148,7 @@ more details, please refer to the [JetS3t Configuration Guide](http://www.jets3t
 | __path__ | The relative path to your files within the bucket |
 | __proxyIdentity__ __proxyCredential__ __proxyPort__ __proxyHost__ | Corresponding parameters if you are accessing the cloud storage provider through a proxy server. |
 | __port__ | Default: 80. The port number through which you want to access Cloud Storage. You should only use the default value unless you need to contact a different endpoint for testing purposes. |
-| __endpoint__ | Default: commondatastorage.googleapis.com. The Cloud Storage hostname. You should only use the default value unless you need to contact a different endpoint for testing purposes. |
+| __endpoint__ | Default: _commondatastorage.googleapis.com_. The Cloud Storage hostname. You should only use the default value unless you need to contact a different endpoint for testing purposes. |
```

Comment:
This isn't what I was talking about. See https://github.com/GoogleCloudPlatform/community/pull/488#discussion_r220281601